app/controllers/places_controller.rb

```diff
@@ -51,17 +51,9 @@ class PlacesController < ApplicationController
 
   before_action :allow_external_iframes, only: [:guide_widget, :cached_guide]
 
-  requires_privilege :organizer, only: [:new, :create, :edit, :update, :destroy],
-    if: proc {| _c |
-      # Only check privileges if the current user didn't create the place, i.e.
-      # allow people to edit places they created even if they haven't earned the
-      # organizer privilege
-      if @place.blank?
-        true
-      else
-        current_user.id != @place.user_id
-      end
-    }
+  requires_privilege :organizer, only: [:new, :create, :edit, :update, :destroy] do
+    redirect_back_or_default(root_url)
+  end
```

Comment:
Why are you removing the ability to edit a place you created regardless of privilege? Also, requires_privilege does not take a block. You can find the full definition at https://github.com/inaturalist/inaturalist/blob/main/lib/privileges.rb#L63

Reply:
@kueda Actually I thought the if block checks whether other users who have not created the place have the edit access. So, now i will keep the 'if block' as it is. But what exactly I have to do for the 2nd Todo task.Can You please explain?

Reply:
@kueda So, I think I should only add ':new' and ':create' in the 'only:' array of the organizer privilege.